webpack/assets/javascripts/react_app/components/FactCharts/FactChart.fixtures.js

```diff
@@ -30,10 +28,3 @@ export const modalLoadingState = Immutable(
     loaderStatus: STATUS.PENDING,
   })
 );
-
-export const modalErrorState = Immutable(
-  Object.assign(state, {
-    modalToDisplay: { 1: true },
-    loaderStatus: STATUS.ERROR,
-  })
-);
```

Comment:
suggestion - if we are already using Immutable library:
- set the `state` object to `Immutable`
- than use its `setIn` functionality instead of `Object.assign`, for example:
```
obj.setIn(['key'], data)
```

Reply:
I have been mistakenly using fixtures in the tests where its no longer needed as the new tests are using snapshots, so most fixtures are removed now

Reply:
Changed, thanks!